config/tileset/aerial.json

```diff
@@ -348,11 +348,11 @@
       "category": "Rural Aerial Photos"
     },
     {
-      "2193": "s3://linz-basemaps/2193/marlborough_2023_0.25m/01H6WRJWDK9VNKK6HE2PZ63AKG/",
-      "3857": "s3://linz-basemaps/3857/marlborough_2023_0.25m/01H6WRKSPQD92366HB8WT35BQK/",
+      "2193": "s3://linz-basemaps/2193/marlborough_2023_0.25m/01J9JDPASA4GCZ6Y6RJMMRD1XH/",
+      "3857": "s3://linz-basemaps/3857/marlborough_2023_0.25m/01J9JDPASBK3P9NSRB6HQN31DQ/",
       "name": "marlborough-2023-0.25m",
       "minZoom": 13,
-      "title": "Marlborough 0.25m Rural Aerial Photos (2023) - Draft",
+      "title": "Marlborough 0.25m Rural Aerial Photos (2023-2024)",
```

Comment:
I am sure why we have `name` and `url` are only for 2023 (marlborough-2023-0.25m) and `title` is 2023-2024?